draft-ietf-taps-interface.md

```diff
@@ -768,15 +768,15 @@ This property specifies whether the application needs or prefers to use a transp
 protocol that preserves message boundaries. The default
 is to Prefer Preservation of Message Boundaries.
 
-### Configure Per-Message Reliability {#prop-partially-reliable}
+### Per-Message Reliability {#prop-partially-reliable}
 
 Name:
 : per-msg-reliability
 
 This property specifies whether an application considers it useful to indicate
 its reliability requirements on a per-Message basis. This property applies to
 Connections and Connection Groups. The default is to Ignore
-this option.
+this Per-Message Reliability.
```

Comment:
I don't like this either - even if we'd go back to the previous name. I agree that "option" doesn't fit though - I think we could either repeat the name as you suggest, but without "this" before it, or we could say "Selection Property". I see that this is a general inconsistency, "this option" appears in multiple places and should be fixed uniformly, everywhere, so it's a good catch, thanks!

Reply:
yes forgot to remove "this". The whole point was consistency. Feel free to take another approach.

Reply:
It's fine without "this", thanks!

Reply:
(and I must remember to fix this uniformly everywhere...)